core/src/main/java/de/danoeh/antennapod/core/service/QuickSettingsTileService.java

```diff
@@ -0,0 +1,69 @@
+package de.danoeh.antennapod.core.service;
+
+import android.content.ComponentName;
+import android.content.Intent;
+import android.os.Build;
+import android.os.IBinder;
+import android.service.quicksettings.Tile;
+import android.service.quicksettings.TileService;
+import android.util.Log;
+import android.view.KeyEvent;
+
+import androidx.annotation.RequiresApi;
+
+import de.danoeh.antennapod.core.preferences.PlaybackPreferences;
+import de.danoeh.antennapod.core.receiver.MediaButtonReceiver;
+import de.danoeh.antennapod.core.service.playback.PlaybackService;
+
+@RequiresApi(api = Build.VERSION_CODES.N)
+public class QuickSettingsTileService extends TileService {
+    /**
+     * Logging tag
+     * Missing final 'e' because of max 23 characters for log tags
+     */
+    private static final String TAG = "QuickSettingsTileSvc";
+
+    @Override
+    public void onTileAdded() {
+        super.onTileAdded();
+        updateTile();
+    }
+
+    // Play/pause playback by faking a Media Button press when clicking the tile
```

Comment:
I wouldn't call this "faking". Sending a control button to the background service is a perfectly valid way of starting playback. I don't think this comment "adds much", you can read what it does a few lines below. So I would remove this as well.

Reply:
Done. FYI this was taken from the comment "Creates an intent which fakes a mediabutton press" on WidgetUpdater.createMediaButtonIntent()